app/controllers/games_controller.rb

```diff
@@ -17,5 +17,6 @@ def create
   def show
     @game = Game.find(params[:id])
     gon.pieces_hash = render_pieces(@game) # call the helper function and get the hash. will be passed onto the view
+    Pusher.trigger('game-#{game.id}', 'refresh_page', {:message => "Your Move!"})
```

Comment:
You'll want `'game-#{game.id}'` to be in double quotes `"game-#{game.id}"`

Ruby only does string concatenation with double quotes. crazy right? :smile: